echo/readme.md

```
@@ -1,175 +1,175 @@
+Service:
+
+- Directus (CMS, Auth Server)
+- Python FastAPI Backend (AI APIs, chat, library, transcription etc)
+
+Clients:
+
+- React Frontends (Admin Dashboard and Participants' Portal, used Directus / Python)
+
+## Getting Started
+
+# How do I run Dembrane ECHO locally?
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document

(MD025, single-title, single-h1)